<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_Edit</class>
 <widget class="QWidget" name="Form_Edit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="userinfo_widget" native="true">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>500</width>
     <height>400</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>500</width>
     <height>400</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>500</width>
     <height>400</height>
    </size>
   </property>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>331</width>
      <height>366</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <item row="2" column="5">
      <widget class="QLabel" name="user5">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>100</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/new/src/img/user5.jpg);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QLabel" name="user4">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>100</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/new/src/img/user4.jpg);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="8">
      <widget class="QLabel" name="user2label">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>22</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
color:rgba(0, 0, 0, 0)</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="4" colspan="5">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="6">
      <widget class="QLabel" name="user4label">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
color:rgba(0, 0, 0, 0)</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="7">
      <widget class="QLabel" name="user2">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>100</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/new/src/img/user2.jpg);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="6">
      <widget class="QLabel" name="user5label">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
color:rgba(0, 0, 0, 0)</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="7">
      <widget class="QLabel" name="user1">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>100</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/new/src/img/user1.jpg);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="8">
      <widget class="QLabel" name="user1label">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
color:rgba(0, 0, 0, 0)</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="8">
      <widget class="QLabel" name="user3label">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
color:rgba(0, 0, 0, 0)</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="7">
      <widget class="QLabel" name="user3">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>100</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/new/src/img/user3.jpg);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="6">
      <widget class="QLabel" name="user6label">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
color:rgba(0, 0, 0, 0)</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="5">
      <widget class="QLabel" name="user6">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>100</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/new/src/img/user6.jpg);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="4" colspan="5">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="backbutton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>90</width>
     <height>180</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/src/img/backbutton.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="sleeplabel">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>430</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color:rgba(0, 0, 0, 0)</string>
   </property>
   <property name="text">
    <string>Sleep</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
